packages/landing/src/components/daterange.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component, ReactNode } from 'react';
+import { Config } from '../config';
+
+const minYear = '1950';
+const maxYear = '2023';
+
+export interface DateRangeState {
+  yearAfter?: string;
```

Comment:
can these be `dateBefore` / `dateAfter` to be consistent with everything else, you can just set them as full ISO strings.